### ISSUE TITLE
New section: plugins & workflows

### ISSUE THREAD

**Issue by julou** [zotero-manual/zotero-manual#14]
Dear authors, 

Thanks for your manual, this was very much needed and it rocks!
Whenever I send the link to colleagues working in natural sciences fields, I need to add things and I thought they would be better off in the manual. In particular, I would happily start to draft a section `Plugins & Workflows` in which plugins would be described in a more detailed manner that in zotero doc (starting with Zotfile and Better Bibtex) and typical worklows would be described in a task oriented manner, e.g.:
- how to import citations from pubmed and attach a pdf
- how to write a markdown document with zotero (Bettr Bibtex export + drag and drop / quick copy)

Please let me know if this sounds relevant to you and/or how to improve it. Of course I can also post such content somewhere else on the web if you prefer.

**Comment by rmzelle** [comment]
I'm a bit ambivalent about this. I agree that documentation on workflows with ZotFile and Zotero Better BibTeX would be valuable to the Zotero community, but I'm wondering if this is the right place.

As you might have noticed, @adam3smith and I haven't been putting a lot of time in this manual recently, since we're rather busy with other (Zotero/CSL) stuff, although we're both still hoping to get back into it. Until we're able to write entries on most of the core topics, I'm hesitant to accept entries on things like plugins.

Any reason you prefer this manual over the Zotero wiki?

**Comment by adam3smith** [comment]
I always figured we'd document ZotFile for sure, so I'd be very happy to take that. The markdown doc @rmzelle and I have to discuss. I'd be inclined to add it because I'm very partial to that as a possibility, but I see why he might be hesitant. It's also an area with lots of possible solutions, so it would either be pretty extensive or we'd have to pick.

**Comment by julou** [comment]
OK, so you discuss and let me know!
@rmzelle I offer to contribute here because this manual is imho of much better quality and much more up-to-date than the wiki. And I ask because I don't work to come up with  pull requests that you think clutter the all thing!

What about sharing typical use cases / workflow? I propose this because I got the impression that the way people use zotero varies from field to field, in particular depending on your sources (jester / scholar, pubmed, etc) and your writing practice (word, latex, markdown, etc + the large choice of editors). So I'd be happy to share what I established as convenient practices (when coming back to zotero after using papers for 3 years).

**Comment by rmzelle** [comment]
Yeah, maybe it would make most sense if you started on the ZotFile documentation?

For Markdown-related documentation, I was under the impression that there was already a lot of it out there, but it seems to be mostly about the CSL/Pandoc combo, e.g. http://programminghistorian.org/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown, http://blog.cigrainger.com/2014/07/pandoc-markdown.html and http://rmarkdown.rstudio.com/authoring_pandoc_markdown.html. So there is probably room for a piece on Zotero/Markdown.

Your English seems pretty good, but do you have any examples of your writing, so we know if you need editing help? I assume you're okay with our CC BY-SA license? Would you be willing to forgo any revenue if we ever end up selling (PDF) copies of the manual (assuming we finish it)? We would probably figure out a distribution in revenue to all large contributors if revenue is ever going to be significant, but for small sums that might not be worthwhile.

**Comment by adam3smith** [comment]
I've added a chapter on add-ons to our outline. 
Feel free to start this in a new md file. 
Note that for screenshots, we always include the Zotero version in the filename to be able to track what may need updating.

**Comment by julou** [comment]
Hello,

As you've seen, other things kept me busy over the past months. I still
hope I'll find some time for this soon!

Best,
Thomas


Le mercredi 20 mai 2015, Sebastian Karcher <notifications@github.com> a
écrit :
